Source/abci/Exporter/aePolyMesh.cpp

```diff
@@ -105,14 +105,11 @@ void aePolyMesh::writeSampleBody()
         ApplyScale(m_buf_velocities.data(), (int)m_buf_velocities.size(), scale);
     }
 
-    // if face counts are empty, assume all faces are triangles
-    if (m_buf_faces.empty())
-    {
-        m_buf_faces.resize((int)(m_buf_indices.size() / 3), 3);
-    }
-
```

Comment:
I'm curious why you removed that and if there is a link with the PR or if it's just a parallel clean?

Reply:
The array was always initialized a bit lower, so code served no obvious purpose.